### DISCUSSION TITLE
Docs: i18n - how to translate routes using app directory?

### DISCUSSION THREAD

**Discussion by branislavbrincko** [vercel/next.js#74497] [App Router]
### What is the improvement or update you wish to see?

The documentation could showcase how to translate routes using app router. For example:

```
/about
/fr/sur
/de/uber
```

### Is there any context that might help us understand?

There have been lengthy discussions how to implement i18n using new app directory:
- https://github.com/vercel/next.js/issues/41980
- https://github.com/vercel/next.js/discussions/18485

Now, there is an official documentation: 
-  https://nextjs.org/docs/app/building-your-application/routing/internationalization

But this documentation, as well as the above mentioned discussions, don't include the _"final"_ guide how to implement translation of page names/urls, for example:
```
/about
/fr/sur
/de/uber
```

(The documentation contains a link to a "minimal example" as well as a link to the next intl library, but neither of these showcase routes translations.)

As this is the crucial SEO element of any multilingual website, it would be nice if the Next.js provided a more detailed guide how to implement this. 

### Does the docs page already exist? Please link to it.

https://nextjs.org/docs/app/building-your-application/routing/internationalization

**Comment by laurenswuyts** [comment]
Any solution?

_Edit by maintainers: Comment was **automatically** minimized because it was considered unhelpful. (If you think this was by mistake, let us know). Please only comment if it adds context to the issue. If you want to express that you have the same problem, use the upvote 👍 on the issue description or subscribe to the issue for updates. Thanks!_

**Comment by hendrik244** [comment]
Still nothing new to this? Also facing this and trying to avoid using third party libraries...
